esiclient/v1/trunk.py

```diff
@@ -0,0 +1,147 @@
+#   Licensed under the Apache License, Version 2.0 (the "License"); you may
+#   not use this file except in compliance with the License. You may obtain
+#   a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+
+import logging
+
+from osc_lib.command import command
+from osc_lib.i18n import _
+
+from esiclient import utils
+
+ACTIVE = 'active'
+ADOPT = 'adopt'
+MANAGEABLE = 'manageable'
```

Comment:
I didn't see them being used in this file. Is there any reason why we have these declared here?

Reply:
Whoops - accidental copy/paste. I'll remove!